client/events.lua

```diff
@@ -23,11 +23,14 @@ end)
 RegisterNetEvent('QBCore:Command:TeleportToPlayer', function(coords)
     local ped = PlayerPedId()
     SetPedCoordsKeepVehicle(ped, coords.x, coords.y, coords.z)
+    exports['qb-doorlock']:updateDoors()
```

Comment:
To decouple the code from specific resources can you please make this broadcast a client event and then add the update doors event  within qb-doorlock.